config.ini-sample

```diff
@@ -1,2 +1,4 @@
 [MapQuest]
 key=your-api-key-goes-here
+[Directory]
+dir=%Y-%m-%b
```

Comment:
FYI, I'm working on an implementation of this. Should have a PR over the weekend.